common/models/load_balancer_summary.go

```diff
@@ -0,0 +1,8 @@
+package models
+
+type LoadBalancerSummary struct {
```

Comment:
I realize these structs are a subset of the structs they're based on, but overall, what are we gaining by having them?